process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
+
+## Archiving Criteria
+
+There are different criteria to consider when archiving a project, but here are ones that the TOC looks for:
+
+* It hasn't seen a commit in over 6 months.
+* It hasn't seen a release in over 6 months.
+* There haven't been any issues opened for 6 months.
+* Opened issues haven't received a response within 6 months.
+* It's binaries/source are no longer being downloaded
+
+It is important to note that there is a difference between a mature project that doesn't get much attention anymore but is stable versus a project that is inactive.
+
+## Voting Process
+
+To archive a project:
+
+* A proposal must be put forth to the TOC repo and be open for at least 2 weeks of discussion.
+* The TOC will inform the CNCF end user community and wider community of all archiving proposals
+* A vote must be finalized with 2/3 approval from the TOC
+
+## Archiving Process
+
+What does archiving for a CNCF project mean?
+
+* CNCF will no longer provide any support for the project, via service desk
+* CNCF will list archived projects online
+* Archived CNCF projects will be transferred to the Linux Foundation for neutral holding and support
```

Comment:
Do we need to say something about the lack of other CNCF support - such as marketing? I'm assuming everything pretty much stops.